src/android.ts

```diff
@@ -310,10 +310,21 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
             // Install app if necessary.
             if (!(await this.isAppInstalled('com.wireguard.android'))) {
                 try {
-                    const fdroidMeta = await fetch('https://f-droid.org/api/v1/packages/com.wireguard.android').then(
-                        (res) => res.json()
-                    );
-                    const apkUrl = `https://f-droid.org/repo/com.wireguard.android_${fdroidMeta.suggestedVersionCode}.apk`;
+                    const fdroidVersionCode = await fetch('https://f-droid.org/api/v1/packages/com.wireguard.android')
+                        .then((res) => res.json())
+                        .then((fdroidMeta) => {
+                            if (
+                                fdroidMeta.packages.some(
+                                    (p: { versionCode: number }) => p.versionCode === fdroidMeta.suggestedVersionCode
+                                )
+                            )
+                                return fdroidMeta.suggestedVersionCode;
+
+                            return fdroidMeta.packages.sort((a: { versionCode: number }, b: { versionCode: number }) =>
+                                a.versionCode === b.versionCode ? 0 : a.versionCode > b.versionCode ? 1 : -1
```

Comment:
You can just subtract the numbers, you don't need to return exactly `1` or `-1`.

Reply:
Also, this chooses the oldest version, which we don't want.

Reply:
Ah, true. It was kinda late when I did that.